index.html

```diff
@@ -187,7 +222,10 @@ <h2 class="h2">Get Started</h2>
                 <textarea type="text" placeholder="Enter your message" name="message" maxlength="500" class="message"  required></textarea>
                 <input type="submit" value="Start Collaboration" class="buttons contact__button">
             </form>
```

Comment:
- [ ] Kindly decrease the `font-size` of the submit button and align it in the center of the form 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200125280-e5bb77af-8926-4ee7-91cc-3f675bd1826d.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200125295-180012fb-8d05-4f8f-8169-681970261fcd.png)

Reply:
- [ Optional ] Kindly decrease the `font-size` of the submit button and align it in the `center` of the form as suggested by the previous reviewer.